spec/models/like_spec.rb

```diff
@@ -1,5 +1,5 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+# RSpec.describe Like, type: :model do
```

Comment:
- [ ] Please add unit tests for your like model as required 

![image](https://user-images.githubusercontent.com/75126481/156415768-14c457ac-ff69-4bb5-8c39-1a61472678da.png)